src/tools/miri/tests/panic/mir-validation.rs

```diff
@@ -1,27 +0,0 @@
-//! Ensure that the MIR validator runs on Miri's input.
```

Comment:
We would still like to be sure that the MIR validator runs in Miri -- we used to have a bug where that didn't happen, which was annoying.

Would be better to not regress test coverage here.